### DISCUSSION TITLE
Addon for invites

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#444] [General]
Good day, is it possible to add an invite addon where people can sign up with a generated invite code?

**Comment by rejetto** [comment]
not really, but since there is no information like "email", "first name" and "last name", what you are asking is basically the same as:
create an account for the person, set a password with the code you call "invitation".
Give them the username and the invitation. They can now login and can change the password.

**Comment by Stoufs** [comment]
Alright, what about disabling account after X amount of days?

**Comment by rejetto** [comment]
i think i will add the possibility to set an expiration date and time for each account, in version 0.52.
how about that?

**Reply by Stoufs** [reply]
Thats great! Can you set it to expire from first login? after x amount of days or minutes or something

**Reply by rejetto** [reply]
i have just to decide if to offer such mechanism as plugin or not

**Reply by Stoufs** [reply]
I see! any progress?

**Reply by rejetto** [reply]
nope

**Comment by rejetto** [comment]
here it is
https://github.com/rejetto/hfs/releases/tag/v0.52.0-alpha1

**Reply by Stoufs** [reply]
This version includes the expiry option?

**Reply by Stoufs** [reply]
Nevermind I see that now it is! Thank you for that! Is it possible to set it to expire on first login after X amount of days?

**Reply by rejetto** [reply]
It's near the expiration field

**Comment by Stoufs** [comment]
Sweet! Is it possible to show the expiration date in the user menu like the quota shows?

**Comment by rejetto** [comment]
like this? in next release
<img width="364" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/cf0f04a0-7361-432a-b63f-c95dd4bd1235">

**Reply by Stoufs** [reply]
That's perfect! As usual, you are awesome! Anyway to get temp release through a link like last time? :)

**Reply by Stoufs** [reply]
For linux :)

**Reply by rejetto** [reply]
You did well at specifying.
Later today

**Reply by Stoufs** [reply]
Much appreciated as always!

**Comment by rejetto** [comment]
[hfs-linux.zip](https://github.com/rejetto/hfs/files/14114641/hfs-linux.zip)
